internal/provider/signin_policy_resource_test.go

```diff
@@ -0,0 +1,42 @@
+package provider
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+const testSignInPolicy string = "aembit_signin_policy.test"
+
+func TestAccSigninPolicy(t *testing.T) {
```

Comment:
Do we need to disable this test on GitHub CI or maybe limit it to prevent the 3 runs from falling on each other?

Reply:
I have added skipCI method for now to make the tests pass, we can later change it if we decide to run it for specific version

Reply:
skipCI worked fine but it ruined the code coverage so I copied getTerraformVersion function from the other PR to run it only for v1.9. All the checks are green now.